include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
If there's no desire to build an out-of-tree provider, then there's no need to install `fi_log.h` (during `make install`).